### PR TITLE
WINC-921: adjust periodic report_template

### PR DESCRIPTION
**Pull request by alinaryan**
Adjusts the community-4.12 and community-4.13 periodic
report_template to include a logs link and commit reference for
clarity and better triaging.

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ tests:
   container:
     from: src
 - as: aws-e2e-operator
-  cron: 0 */4 * * *
+  cron: 0 0 * * *
```
Comment (saifshaikh48) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml:56]: thoughts on making these only Monday-Friday?
`0 0 * * 1-5`

Comment (alinaryan) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml:56]: I'd be ok with that

Comment (alinaryan) [reply ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.13__okd.yaml:56]: I will do this in the next PR.. just going to make sure this version of the template works for now. [Slack thread ](https://redhat-internal.slack.com/archives/CM4ERHBJS/p1676578098136429)

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Comment by mansikulkarni96** [comment]
LGTM

**Review by sebsoto — APPROVED** [review]

**Comment by alinaryan** [comment]
/pj-rehearse periodic-ci-openshift-windows-machine-config-operator-community-4.12-okd-aws-e2e-operator

**Pull request merged**